charts/keycloak/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: keycloak
-version: 9.9.5
+version: 9.9.6
```

Comment:
Please bump the minor version for a new feature to follow semver.
```suggestion
version: 9.10.0
```

Reply:
minor version bumped as requested